dotCMS/src/curl-test/Workflow Resource Tests.json

```diff
@@ -14784,17 +14785,166 @@
 					"response": []
 				},
 				{
-					"name": "update-contentlet-remove-category",
+					"name": "update-contentlet-add-category-ignore-fields",
```

Comment:
Could you please add a description to the new tests implemented here?

Reply:
done!